src/components/UIComponent/ApparatusInfo.tsx

```diff
@@ -25,9 +21,7 @@ function ApparatusInfo({experienceData,addActionToList, removeActionFromList}: O
       />
       <ActionBox
         assetBundle={assetBundle}
-        addAction={(actionData) =>
-          addActionToList(actionData)
-        }
+        addAction={(actionData) => addActionToList(actionData)}
```

Comment:
since we are using the context, this prop can be removed and in actionbox add action to list can be used

Reply:
Changes have been made. Thank you for the comment.

Reply:
did you push it? I just pull the newest version and my vscode stills gives me an error with addAction
Type '{ assetBundle: { children: any[]; path: string; identifier: string; }; addAction: (actionData: any) => void; }' is not assignable to type 'IntrinsicAttributes & ActionBoxProps'.
  Property 'addAction' does not exist on type 'IntrinsicAttributes & ActionBoxProps'.